server/src/main/scala/org/http4s/server/middleware/URITranslation.scala

```diff
@@ -4,6 +4,7 @@ package middleware
 
 import cats.Functor
 
+@deprecated("Use org.http4s.server.middleware.UriTranslation instead", since = "0.18.16")
```

Comment:
Do you mean `TranslateUri`?

Reply:
Good catch.  If only we could compile our deprecation messages.

Reply:
Yeah, I tried just changing the case, but got a compiler warning.